### ISSUE TITLE
Dotnet New Template FAKE + Paket

### ISSUE THREAD

**Issue by TclasenITVT** [fsharp/FAKE#2145]
### Description

I have just set up my first project to use Paket + FAKE. It took me some time to get it going even with the FAKE-template that is available as it was my understanding that I would still need to set up Paket myself. If this is an error on my end that the template works without .paket-folder and doesn't require the steps seen on the Paket website then you should maybe improve the documentation a bit on how to get startet with the template. 
If not then creating a combined template would be really nice thing as this would speed up getting it to work.

As this is more of a question/feature request in the broader sense I don't think I would need to fill out the rest of the points in your issue template.

**Comment by matthid** [comment]
Either with or without paket are both supported usage scenarios. Can you give an example on what we should change where?

**Comment by TclasenITVT** [comment]
Create a second dotnet new template that additionally contains the .paket-folder which contains the paket-bootloader.exe renamed to paket.exe as described in the [paket documentation](https://fsprojects.github.io/Paket/paket-and-dotnet-cli.html#Downloading-Paket-s-Bootstrapper).
Also a short usage manual included with the template would be nice. Something that give a little bit more details of how to get started with the template and what the usecases are.

**Comment by matthid** [comment]
I'd rather focus on the 'without' paket workflow until paket runs on netcore. Any idea what we should change in the docs/template?

**Comment by matthid** [comment]
As discussing on another issue I feel like the problem is that the ootb template seems to have a problem when using the default settings, and we maybe should change the default to `inline` /cc @kblohm

**Comment by kblohm** [comment]
As for adding paket, i would guess that most people that use FAKE are also using paket, so that might be a nice convenience. But then again, with the global tool for paket it should be fairly easy to install that for a new project. Not sure whats the best approach there. I feel like the templating/setting up projects for F# is lacking/ not very beginner friendly in general but that is probably another discussion

What is the problem with the default settings?
As for the `inline` dependencies: i prefer the `paket.dependencies` approach. When i am using paket anyways, i see no real benefit in using the inline dependencies. Even in projects where i use nuget, i add paket just for FAKE. I prefer running `paket install` instead of deleting the `build.fsx.lock` file. That seems more intuitive to me. I also find it "easier" to install tools, e.g. `nuget.exe`, with the dependency file, because that uses the default `packages` folder that most of the FAKE-Modules expect. But that is obviously personal preference.

**Comment by matthid** [comment]
@kblohm I'm fine either way but I feel like whatever the defaults are they should work without error and without any paket.

Afaik when `paket.dependencies` exists fake will just create a `paket.lock` file on demand but apparently that doesn't work?

@TclasenITVT I think we need some clarification what exactly you had problems with. I just tried the following:

```
dotnet new fake
.\fake.cmd build
```

and it worked (it created `paket.lock` which you can delete similar to `build.fsx.lock` in order to update). So no paket is required

**Comment by TclasenITVT** [comment]
I just didn't know that it works like that. Maybe just add that to the gettingstarted page that you should just run i like you said?

**Comment by MikaelUmaN** [comment]
@matthid 

In my case I next wanted to use the Fake.JavaScript.Npm module. For this I had to add the nuget package. But how do I add that and make it available in my build script?

Only adding to paket.dependencies and trying to build again did nothing. I tried to add --nocache as an argument. Still nothing. I ended up installing paket and doing a complete restore, at which point I could proceed.

So, unless I am missing something I think the template kind of makes you dependent on Paket, which I don't think should be that necessary anymore given the improvements in nuget and dotnet cli.
